src/Alert/index.js

```diff
@@ -0,0 +1,151 @@
+// @flow
+import * as React from "react";
+import styled from "styled-components";
+import defaultTokens from "@kiwicom/orbit-design-tokens";
+import { withTheme } from "theming";
+
+import {
+  InformationCircle,
+  Check,
+  Alert as AlertTriangle,
+  AlertCircle,
+  CloseCircle,
+} from "../icons";
+import { Button } from "../index";
+
+const Icon = ({ icon, type }) => {
+  let renderIcon;
+  if (typeof icon === "boolean") {
```

Comment:
icon as boolean, really ?

Reply:
It shouldn't be a problem because when you type `<Alert type="info" icon />` it returns you `true`, without the icon, it returns you `false`. When you want to pass there some specific icon, you can do it like `<Alert type="info" icon="<Airplane />" />` and this returns you a `React.Element`. So you can define it in flow like `icon?: React.Element<any> | boolean`